salt/engines/__init__.py

```diff
@@ -43,7 +43,7 @@ def start_engines(opts, proc_mgr):
 
     for engine in engines_opt:
         if isinstance(engine, dict):
-            engine, engine_opts = engine.items()[0]
+            engine, engine_opts = list(engine.items())[0]
```

Comment:
I am a little confused. We call `.items()` , not `.keys()` .
I think the logic wants the first item (not key) in the dict called `engine`.